content/glossary.md

```diff
@@ -3,19 +3,52 @@ title: Glossary
 kind: Documentation
 draft: true
 ---
+
 **Agent**
 
+**API**
+
+Datadog provides an HTTP API in order to interact programmatically with the platform. Every feature, resource, mechanism, and bit of functionality can be accessed via the API. Interacting with the API manually is a great way to learn how Datadog works under the hood! In production however, you will use a purpose-built tool, library, or interface — such as the Datadog product itself — as an abstraction layer between you and the API.
+
+See the [API documentation][1] for more information.
+
 **Check**
 
+Checks are small Python programs run periodically by the Agent. A Check performs an action and then gathers the result, which the Agent then stores and reports to the Datadog platform. These programs are freeform and are generally used to collect metrics from custom environments or applications.
+
+Note that the word “check” — when not capitalized — refers to the generic act of taking a measurement.
```

Comment:
or maybe this is a change that was made in the last few years that I never noticed.